modules/ml_models/scikit_online_learners.py

```diff
@@ -0,0 +1,33 @@
+from sklearn import linear_model
+from modules.toolbox import framework_tools as ft
+
+#
+# There is also some repetitive code in terms of fitting and scoring models, this will
+# likely be split out into a new helper
+#
+
+def run_sgd_regressor(path, filename):
```

Comment:
Maybe add a reference here to what scikit method this is hitting so one can learn about the actual method/algorithm. Could also help in the other direction where one who knows the scikit methods can easily find the right method in these files.